.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug
+about: Create a bug report to help us improve
+title: '[BUG]'
+labels: kind/bug
+assignees: ''
+
+---
+
+ **Describe the bug**
+A clear and concise description of what the bug is.
+
+ **To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2.
+3.
+
+ **Expected behavior**
+A clear and concise description of what you expected to happen.
+
+ **Screenshots**
```

Comment:
I'd omit this -- it seems unlikely that most screenshots will be more useful than e.g. console dumps into code-fenced blocks.